app/modules/intelligence/agents/chat_agents/tool_helpers.py

```
@@ -172,5 +179,15 @@ def get_tool_result_info_content(tool_name: str, content: List[Any] | str | Any)
 """
                 return res[: min(len(res), 600)] + " ..."
             return ""
+        case "fetch_file_content":
+            print(content)
+            if isinstance(content, Dict):
+                if content.get("success") == False:
```

Ruff (0.8.2)

Avoid equality comparisons to `False`; use `if not content.get("success"):` for false checks

Replace with `not content.get("success")`

(E712)